components/templates/src/global_fns/load_data.rs

```diff
@@ -23,6 +24,11 @@ enum DataSource {
     Path(PathBuf),
 }
 
+enum Method {
```

Comment:
For enums like that, derive automatically the basic Debug/PartialEq/Clone/Copy it's almost free

Reply:
implemented in the latest commit